.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file, `.prettierrc` and `package.json` are the only manual changes. All other changes are coming from Prettier.